bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/HadoopFileSource.java

```diff
@@ -0,0 +1,534 @@
+/*
+ * Copyright (C) 2015 The Google Cloud Dataflow Hadoop Library Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.bigtable.dataflow;
```

Comment:
Maybe we can leave it as is for now, until after we contribute our code to dataflow?